common.gypi

```diff
@@ -269,7 +269,8 @@
           ['clang==1', {
             'xcode_settings': {
               'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0',
-              'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++0x',  # -std=gnu++0x
+              'CLANG_CXX_LANGUAGE_STANDARD': 'c++11',  # -std=c++11
+              'CLANG_CXX_LIBRARY': 'libc++', #-stdlib=libc++
```

Comment:
This is libc++, not libstdc++, and it's for mac only. On other platforms it compiled as is, so I doubt it added any new requirement to libstdc++.